docs/src/main/sphinx/admin/properties-writer-scaling.rst

```diff
@@ -35,9 +35,20 @@ divided by the number of writers per task. This can be specified on a
 per-query basis using the ``writer_min_size`` session property.
 
 ``use-preferred-write-partitioning``
-^^^^^^^^^^^^^^^^^^^
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 * **Type:** ``boolean``
 * **Default value:** ``false``
 
 Enables preferred write partitioning. This option will force every partition
 to be written by a single worker.
+
+``preferred-write-partitioning-min-number-of-partitions``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* **Type:** ``integer``
+* **Default value:** ``50``
+
+The minimum number of partitions required to trigger the ``preferred write
+partitioning`` optimisation. If the number of partitions cannot be estimated
```

Comment:
`if the number of partitions cannot be estimated or preferred-write-partitioning-min-number-of-partitions > 1 then preferred write partitioning won't be used. Setting preferred-write-partitioning-min-number-of-partitions to value less or equal to 1 will force Trino to use preferrered write partitioning`

Reply:
`or preferred-write-partitioning-min-number-of-partitions > 1 then preferred write partitioning won't be used` means that setting this value > 1 will always disable the feature. And this is not the case.
Let's wait for @mosabua to comment on that.